forgerock-auth/src/main/java/org/forgerock/android/auth/CookieMarshaller.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Do you think it would fit on 'core' module?

Reply:
For now, I would keep in in the auth module, those classes are internal we can easily move it to core when necessary.